apps/service_providers/llm_service/runnables.py

```diff
@@ -199,6 +200,8 @@ def prompt(self):
 
 
 class AssistantExperimentRunnable(BaseExperimentRunnable):
+    input_key = "content"
```

Comment:
Nope, it doesn't change anything for the user. This is needed because the assistant runnable expects input as `{"content": prompt}` whereas the other runnables expect `{"input": prompt}`. I don't know why they are different.